posts/kute-zelazo-vs-aluminium copy.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Jak działają bramy przesuwne?"
+date: "April 8, 2021"
```

Comment:
why this date?

Reply:
please use newer date, it will look more convincing and up-to-date if someone open the page